network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/aside.html

```diff
@@ -0,0 +1,18 @@
+{% load wagtailcore_tags %}
+
+{% block block_head %}{% endblock %}
+
+<div class="tw-container {{block.block_type}}-block {% block block_container_classes %}{% endblock %} tw-border-t tw-border-black tw-py-2">
```

Comment:
**Question** Do we need the `tw-container` and the nested `tw-row`?

Reply:
Thanks for questioning these, as above, I've cleaned this up in 0a7d97582601fab2407d5acf995981a73c381bef

I was of the opinion the template should match the other block templates and provide the same overrides and markup but really none of them are needed so I've removed them